frontend/src/metabase/components/form/FormMessage.styled.jsx

```diff
@@ -0,0 +1,21 @@
+import styled, { css } from "styled-components";
+
+import { space } from "metabase/styled-components/theme";
+import { color } from "metabase/lib/colors";
+
+export const FormMessageStyled = styled.span`
+  color: ${({ hasSucceeded }) =>
+    hasSucceeded ? color("success") : color("error")};
+  float: left;
```

Comment:
Why `float`?
I'm afraid it might come back and bite us.

This span is inside a `div` and that parent div is not cleared (it has a height of 0).

Reply:
@nemanjaglumac, I looked into this and came out thinking we're better off with a float than what we have or had.

See the visual example of the Google auth above.

The error message is very long, and with long copy, keeping it to the right as we had before is prone to strange or broken layouts.

Also, the wrapping div is a sign that this is fragile, right? To depend on people knowing to add a container element, and to treat them in a particular way.

It's a good thing we are developing proper components for this soon. 

To sum it up, the way we do it here seems to make the error messages just land above the submit buttons. Unless I'm missing something, I think it's an acceptable solution until we get the components.

Reply:
If it's a temporary solution, so be it.
It currently looks better than before. I'm sure we could find some edge cases where `float` would backfire if we poke at it long enough. But it's pointless to invest much time in something that's not such a high priority.